src/metabase/api/task.clj

```diff
@@ -0,0 +1,26 @@
+(ns metabase.api.task
+  "/api/task endpoints"
+  (:require
+    [compojure.core :refer [GET]]
+    [metabase
```

Comment:
When there's just one namespace, you are importing, you can use:
```
[metabase.util :as u]
```
Separating it like you're doing here used more when you're importing more than one, i.e.
```
[metabase
  [schema :as s]
  [util :as u]]
```